core/food_apis/update_manager.py

```
@@ -524,6 +552,113 @@ async def _update_off_database(self, force: bool = False) -> UpdateResult:
+                        file_path = matching_files[0]
+                        cache_data = {}
+                        with file_path.open("r", encoding="utf-8") as f:
+                            for line in f:
+                                try:
+                                    data = json.loads(line.strip())
+                                    if "name" in data:
+                                        cache_data[data["name"]] = data
+                                except json.JSONDecodeError:
+                                    continue
+                        return cache_data
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)